packages/x-date-pickers-pro/src/internals/models/dateRange.ts

```diff
@@ -14,6 +14,8 @@ import { RangeFieldSection } from './fields';
 export interface DayRangeValidationProps<TDate> {
   /**
    * Disable specific date.
+   *
+   * Warning: This validation will run on every rendered date. Expensive computations can impact performance.
```

Comment:
Could we add this to the regular Date validation type? 🙈 👌

Reply:
Did you mean `DayValidationProps`?
And also did you mean to also add this, or just move this warning to there? 😕 😅

Reply:
Yup, `DayValidationProps`.
Add, this warning is relevant to both range and single pickers alike. 👌

Reply:
Thought so! Thanks for the clarification! 🙇🏼